src/style.ts

```diff
@@ -0,0 +1,16 @@
+import cssRules from "./style.css";
```

Comment:
```suggestion
import cssRules from "./style.css" with { type:'text' };
```

With this we shall be able to remove the custom plugin.

Reply:
Oh you are right, I though it was already implemented.